perception/sub8_perception/sub8_vision_tools/rviz.py

```diff
@@ -14,8 +14,8 @@ class RvizVisualizer(object):
     '''Cute tool for drawing both depth and height-from-bottom in RVIZ
     '''
 
-    def __init__(self):
-        self.rviz_pub = rospy.Publisher("visualization/projection", visualization_msgs.Marker, queue_size=3)
+    def __init__(self, topic="visualization/projection"):
```

Comment:
This is just the default. Individual nodes (like buoy_finder) will override it. I think we should go through and get all perception nodes publishing to the same rviz marker topic later